spec/lib/koala/api/graph_batch_api_spec.rb

```diff
@@ -618,7 +618,7 @@
 
     describe "binary files" do
       it "posts binary files" do
-        file = File.open(File.join(File.dirname(__FILE__), "..", "fixtures", "beach.jpg"))
+        file = File.open(File.join(File.dirname(__FILE__), "../../..", "fixtures", "beach.jpg"))
```

Comment:
Not the most elegant solution. At the very least, I could probably extract this into a helper method.